# Mar - Andrew Clark

**Speaker**: Andrew Clark

[Download Seminar Poster :material-file-pdf-box:](http://www.vanbug.org/wp-content/uploads/2011/02/poster_mar_10_2011.pdf "poster_mar_10_2011.pdf")

**Presentation:** [Download Presentation:material-file-pdf-box:](https://drive.google.com/file/d/1CWlENAUl_BKag58bK41NQHmr_GVLFi87/view?usp=sharing)

[Download Movie ![FLASH](/images/flash.gif)](http://www.vanbug.org/talk_ppts/2010-11/201103_AndrewClark/VanBUG_Mar102011.html) \*Note: There is a problem with the sound for the second half of the talk.

**Talk Title:** Human genomics with large sample size and full genome sequences

!!! info "Event Details"
    
    
    **Date/Time:**
    
    Thursday, March 10, 2011, 6:00 pm

**Affiliation:** Professor of Population Genetics, [Department of Molecular Biology and Genetics](http://mbg.cornell.edu/), Cornell University

**URL:** [Andrew G. Clark](http://mbg.cornell.edu/cals/mbg/research/clark-lab/index.cfm)

**Abstract:**

The 1000 Genomes Project has completed its pilot phases including (1) low-coverage whole-genome sequencing of 179 individuals, (2) high-coverage sequencing of two mother–father–child trios; and (3) exon-targeted sequencing of 697 individuals from seven populations. These data present an unprecedented level of detail about the allele frequency spectrum and local haplotype structure of ~15 million SNPs, 1 M indels, and 20,000 structural variants across the human genome. On average, each person is found to carry approximately 250 to 300 loss-of-function variants in annotated genes. Data from the two trios produced an estimate of 1028 germline mutations per genome per generation. These data also present an opportunity to test for past signatures of natural selection and demographic effects. In a separate study we examined sequence variation in just two genes from the ARIC cohort study of 13,715 people. This analysis resulted in nearly 4x the expected count of singleton variants which were subsequently confirmed. These data are consistent with a very recent explosive population growth model that matches the historical and archeological record. The data are also consistent with the observation that each new full human genome that is sequenced typically finds upwards of 100,000 novel SNPs that are not yet in any database.

**Please note:**

Trainees are invited to meet with the VanBUG speaker for open discussion

of both science and career paths. This takes place 4:30-5:30pm in either

the Boardroom or Lunchroom on the ground floor of the BCCRC

**Recommended Readings:**

[Human genome variation](http://www.ncbi.nlm.nih.gov/pubmed/20981092)

[Deep resequencing](http://www.ncbi.nlm.nih.gov/pubmed/21119644)

---

**Introductory Speaker:** Rodrigo Goya, PhD Candidate, Marra Lab, Genome Science Centre, BCCA & Meyer Lab, Centre for High-Throughput Biology (CHiBi), UBC

**Title:** Characterizing alternative expression profiles in cancer samples using RNA-Seq

