# Apr - Robin Dowell

**Speaker**: Robin Dowell

**Talk Title:** “Enhancer RNA Profiling Predicts Transcription Factor Activity”

![type:video](https://www.youtube.com/embed/ZUKPSS9V0KI)

!!! info "Event Details"
    
    
    **Date/Time:**
    
    Thursday, April 6, 2017 6:00pm

**Affiliation:** Assistant Professor, BioFrontiers Institute and Molecular, Cellular, and Developmental Biology at the University of Colorado in Boulder

**Web-site:** [Robin Dowell](http://dowell.colorado.edu/people.html)

**Twitter:** [@rddowell](https://twitter.com/rddowell)

**Abstract:**

Transcription factors (TFs) exert their regulatory influence through the binding of enhancers, resulting in coordination of gene expression programs. Active enhancers are often characterized by the presence of short, unstable transcripts termed enhancer RNAs (eRNAs). While their function remains unclear, we demonstrate that eRNAs are a powerful readout of TF activity. We infer sites of eRNA origination across hundreds of publicly available nascent transcription datasets and show that eRNAs initiate from sites of TF binding. By quantifying the co-localization of TF-binding motifs and eRNA origins, we derive a simple statistic capable of inferring TF activity. In doing so, we uncover dozens of previously unexplored links between diverse stimuli and the TFs they affect. Our approach constitutes a fundamentally unique strategy for discovering links between TF activity and phenotypes.

**Bio:**

Robin Dowell is an Assistant Professor in the BioFrontiers Institute and Molecular, Cellular, and Developmental Biology at the University of Colorado in Boulder. Using a combination of machine learning and genome biology techniques, the Dowell laboratory seeks a mechanistic understanding of transcription regulation. To this end, the Dowell lab leverages a variety of experimental and computational approaches. Our recent focus has been on enhancers and their transcripts (eRNAs) which we detect and study by leveraging true measures of nascent transcription such as global run-on sequencing (GRO-seq). They have utilized the GRO-seq protocol to gain insight in the behavior of the tumor suppressor transcription factor p53 (Allen et. al. 2014) and CDK8-Mediator (Galbraith et. al. 2013). More recently they have significantly improved the GRO-seq protocol (Allen et. al. in prep.) and developed numerous improved algorithms for GRO-seq analysis (Azofeifa et. al. 2014; Azofeifa et. al. 2016; Lladser et. al. 2017; Azofeifa et. al. in review). Their work shows that eRNAs originate from active transcription factor binding sites. Furthermore, they leverage the pattern of eRNA usage to infer which transcription factors are currently active in any particular cell type. Finally, they have demonstrated the ability to identify the key initial transcriptional responders upon drug treatment.

**Please note:**

Trainees are invited to meet with the VanBUG speaker for open discussion of both science and career paths. This takes place 5:00-5:45pm in either the Boardroom or Lunchroom on the ground floor of the BCCRC

**Recommended Readings**

---

**Introductory Speaker:** Yen-Yi Lin (PhD student, Dr. Cenk Sahinalp’s lab, SFU)

**Title:** “Discovery and genotyping of novel sequence insertions in many sequenced individuals”

**Webcast Link:** <http://vidyoreplay.computecanada.ca/replay/webcastShow.html?key=TAKdbjoDvIqKdLn>

(This technology is brought to you by Compute Canada and WestGrid with support from PHSA Telehealth)

