# Nov - Quaid Morris

**Speaker**: Quaid Morris

[Download Seminar Poster :material-file-pdf-box:](http://www.vanbug.org/wp-content/uploads/2010/11/poster_nov_2010.pdf "poster_nov_2010.pdf")

**Presentation:** [Download Presentation:material-file-pdf-box:](https://drive.google.com/file/d/1wECdttvALuuXuIWoqbZShZFQBk7snT8O/view?usp=sharing)

[Download Movie ![FLASH](/images/flash.gif)](http://www.vanbug.org/talk_ppts/2010-11/201011_QuaidMorris/VanBUG_Nov2010.html)

**Talk Title:** Predicting the targets of mRNA-binding proteins (and a bit of GeneMANIA)

!!! info "Event Details"
    
    
    **Date/Time:**
    
    Wednesday, November 10, 2010, 6:00 pm

**Affiliation:** Quaid Morris is an assistant professor in the Donnelly Centre at the University of Toronto in Canada. He is a computational biologist whose doctoral training was in machine learning at MIT and the Gatsby Unit in London. His lab uses statistical learning to make biological discoveries. Right now, he is interested in post-transcriptional regulation, automated prediction of gene function, and understanding cancer (and other complex diseases) using genomics.

**URL:** [Quaid Morris](http://morrislab.med.utoronto.ca/)

**Abstract:**

RNA-binding domains are among the most common domains in eukaryotic genomes and RNA-binding proteins (RBPs) play critical roles in post-transcriptional regulation (PTR) of gene expression by regulating mRNA processing, mRNA translation, mRNA export and mRNA stability. Many RBPs bind in a sequence-specific manner however, their sequencing binding preferences alone are insufficient to uniquely identify their targets.

As a first step towards building quantitative models of PTR, we are mapping out mRNA and RBP interactions using a combined biochemical and computational strategy. Our strategy is based on a microarray-based assay, called RNAcompete, that measures the binding affinity of a recombinant RBP for hundreds of thousands of short RNA sequences. These sequences are designed to comprehensively query the space of possible binding preferences. We use a new RNA motif finding algorithm, RNAcontext, to infer sequence and structural binding preferences of RBPs from both in vitro RNAcompete data, as well as, in vivo binding data from large-scale immunoprecipation-based assays. Using these motif models to find RBP binding sites on mRNAs requires estimating mRNA secondary structure computationally. I will present some recent work that suggests that estimating this structure is easier than expected.

I will also say a few words about the GeneMANIA project. We have collected more than 800 interaction networks for six major model organisms that you can used to analyze and extend gene lists using either a user-interface web interface (http://www.genemania.org) or a Cytoscape plugin. By querying this interaction data using a fast and accurate gene function prediction algorithm, GeneMANIA can tell you how genes in your list are connected to one another, find more genes like those on your list and possibly what type of data you should collect to help extend your list. (GeneMANIA is collaborative work with Gary Bader).

**Please note:**

Trainees are invited to meet with the VanBUG speaker for open discussion

of both science and career paths. This takes place 4:30-5:30pm in either

the Boardroom or Lunchroom on the ground floor of the BCCRC

**Recommended Readings**

[RNAcompete](http://www.ncbi.nlm.nih.gov/pubmed/19561594)

[RNA binding protein motifs](http://www.ncbi.nlm.nih.gov/pubmed/20418358)

[RNAcontext](http://www.ncbi.nlm.nih.gov/pubmed/20617199)

---

**Introductory Speaker:** Ryan Morin, PhD Candidate, Marra Lab, Genome Sciences Centre, BCCA

**Title:** Mining RNA-seq, Exome and Genome Data for Novel Cancer Driver Mutations

**Presentation:** [Download Presentation:material-file-pdf-box:](https://drive.google.com/file/d/1buP9tKO2OIxYGA1YODl0Ekk0ZLrXZAAu/view?usp=sharing)

