# Sep - Sarah Teichmann

**Speaker**: Sarah Teichmann

### Co-sponsored with

[![MITACS](/images/mitacs.gif)](http://www.mitacs.math.ca)

[Download Seminar Poster :material-file-pdf-box:](https://drive.google.com/file/d/1KNH07xtK9vq6Jdvukvon1gOF9tbPw9Kc/view?usp=sharing)

**Talk Title:** Decoding genetic switches in T helper cell differentiation

!!! info "Event Details"
    
    
    **Date/Time:**
    
    Thursday, September 9, 2010, 6:00 pm

**Affiliation:** [Theoretical and Computational Biology, MRC Laboratory of Molecular Biology](http://www.mrc-lmb.cam.ac.uk/tcb/) , Cambridge University, UK

**URL:** [Sarah Teichmann](http://www.mrc-lmb.cam.ac.uk/genomes/Sarah/)

**Abstract:**

Gene expression levels are believed to be continuously distributed from very low to very high levels, with most genes at an intermediate level. We have studied the transcriptome-wide distribution of expression levels in mouse T helper cells using RNA-seq technology, and have developed a variety of ways to model the expected background levels. This is critical for definition of a threshold level of expression of a gene in a given cell type. In order to calibrate the RNA-seq expression levels in terms of molecules of mRNA per cell, we have integrated the RNA-seq data with single molecule mRNA-FISH experiments.

The results of these analyses and experiments show that many genes are expressed at > ~ 1 molecule per cell and that two major expression levels can be identified which vary by roughly one to two orders of magnitude. This gives rise to bimodal distributions of gene expression levels in cell populations. Analysis of histone modifications by ChIP-seq indicates that activating modifications such as H3K9/14ac and H3K4me3 are involved in this ‘digital’ expression switch.

Our findings have broad implications for the analysis RNA-seq and ChIP-seq data and for the understanding of the regulation of gene expression.

**Please note:**

Trainees are invited to meet with the VanBUG speaker for open discussion

of both science and career paths. This takes place 4:30-5:30pm in either

the Boardroom or Lunchroom on the ground floor of the BCCRC

**Recommended readings:**

[Transcription factors](http://www.ncbi.nlm.nih.gov/pubmed/19274049)

[Structural Bioinformatics](http://www.ncbi.nlm.nih.gov/pubmed/18563089)

---

**Introductory Speaker:** Anthony Fejes, PhD Candidate, Jones Lab, Genome Sciences Centre, BCCA

**Title:** Harnessing Diversity: Leveraging a Database of Human Genome Variations to Tackle Genetic Diseases.

