# Jan - Gary Bader

**Speaker**: Gary Bader

[Download Seminar Poster :material-file-pdf-box:](https://drive.google.com/file/d/14FOYqJSUBQSBhek52Jt62wnkk6A3Xfla/view?usp=sharing)

**Presentation:** [Download Presentation:material-file-powerpoint-box:](https://drive.google.com/file/d/146olMh6s5xpzaVe3pHMFr6ZdKMyFt-kf/view?usp=sharing)

[Download Presentation Movie ![FLASH](/images/flash.gif)](http://www.vanbug.org/talk_ppts/2008-9/Gary_Bader_Jan2009/Gary_Bader_Jan2009.html)

**Talk Title:** Predicting PDZ domain protein interactions from the genome

!!! info "Event Details"
    
    
    **Date/Time:**
    
    January 8, 2009, 6:00pm

**Affiliation:** [University of Toronto](http://www.utoronto.ca/)

**URL:** [Gary Bader](http://baderlab.org/GaryBader)

**Abstract:**

Eukaryotic cell signaling networks, often misregulated in disease, are formed by precisely controlled specific molecular interactions. These interactions are difficult to experimentally map, thus it would be ideal if they could be predicted directly from the genome sequence. Unfortunately, this is not currently possible. Interestingly, a significant fraction of these are protein-protein interactions mediated by peptide recognition modules reused in different combinations to create many proteins with different functions. These domains are relatively easy to detect in genes, through sequence similarity to known family members. A complete specificity map of peptide recognition preferences for all of these domains would enable accurate prediction for a significant subset of protein-protein interactions directly from the genome. Advances in DNA sequencing, phage display and protein chips, have led to large-scale binding preference mapping for some of these domains and expression and localization profiles that map when and where a given protein may exist in a specific cell type are also available. We are developing computational methods to combine and use this data to accurately predict biologically relevant protein interactions involving PDZ, WW and SH3 domains for experimental validation directly from a genome sequence. This will contribute to the understanding of diverse cell signaling networks, will reduce the effort required to map molecular interaction networks, will enable de novo network

engineering using synthetic biology methods and will allow accurate prediction of the functional consequence of genetic mutations associated with disease.

---

**Introductory Speaker:** ### Jesse Gillis

**Affiliation:** [Pavlidis Lab, UBC Centre for High-througput Biology](http://www.bioinformatics.ubc.ca/pavlidis/lab/)

**Presentation:** [Download Presentation:material-file-powerpoint-box:](https://drive.google.com/file/d/1dJx1Zs7nbqJhMR_BTCJO4H0gXAOdAiCx/view?usp=sharing)

**Talk Title:** A meta-analysis of differential coexpression across age

