# Dec - Rob Beiko

**Speaker**: Rob Beiko

**Talk Title:** “A Coevolution-Based Approach to the Identification of Genes with Similar Phylogenetic Distributions”

![type:video](https://www.youtube.com/embed/ui6mPJteXfk)

!!! info "Event Details"
    
    
    **Date/Time:**
    
    Thursday, December 15, 2016 6:00pm

**Affiliation:** Associate Professor, Faculty of Computer Science, Dalhousie University

Canada Research Chair in Bioinformatics

**Web-site:** [Rob Beiko](http://kiwi.cs.dal.ca/beikolabWordPress)

**Twitter:** [@rob\_beiko](https://twitter.com/rob_beiko)

**Abstract:**

Homologous genes in prokaryotes can be described using phylogenetic profiles which summarize their patterns of presence or absence across a set of genomes. Phylogenetic profiles have been used for nearly twenty years to cluster genes based on measures such as the Euclidean distance between profile vectors. However, most approaches do not take into account the phylogenetic relationships amongst the profiled genomes, and overrepresentation of certain taxonomic groups (for example, intensively sequenced pathogens) can skew the interpretation of profiles. We propose a new approach that uses a coevolutionary method defined by Pagel to account for the phylogenetic relationships amongst target organisms, and a hierarchical clustering approach to define sets of genes with common distributions. Our simple hierarchical approach outperforms more-sophisticated clustering algorithms, and the clusters we obtain show greater evidence of phylogenetic and functional clustering than another recently published approach. Clusters we identified through our approach include amino-acid biosynthesis genes that constitute cohesive pathways, and motility/chemotaxis genes with common histories of descent and lateral gene transfer.

**Bio:**

Robert Beiko is a Canada Research Chair in bioinformatics. His PhD research at the University of Ottawa used machine-learning methods to identify DNA sequences that control the expression of genes. He did a postdoctoral fellowship at the Institute for Molecular Biosciences at the University of Queensland, and performed the first large-scale mapping of lateral gene transfer – the “sharing” of genes, including virulence factors and antimicrobial resistance genes, among microbes. Since taking up his faculty position in 2006 his lab has developed software for the analysis of microbial genomes and microbial communities, and more recently for the analysis and application of next-generation sequencing data for functional trait identification and conservation genetics. He has also contributed to the bioinformatics community in Canada, serving on the Bioinformatics and Computational Biology National Strategy Committee, and co-organizing the first Canadian Bioinformatics Workshop in Metagenomic Data Analysis.

**Please note:**

Trainees are invited to meet with the VanBUG speaker for open discussion of both science and career paths. This takes place 5:00-5:45pm in either the Boardroom or Lunchroom on the ground floor of the BCCRC

**Recommended Readings**

---

**Introductory Speaker:** Maia Smith (MSc, Dr. Sohrab Shah’s lab, BCCRC)

**Title:** “Visualization methods for spatial and temporal clonal evolution in cancer.”

**Webcast Link:** <http://vidyoreplay.computecanada.ca/replay/webcastShow.html?key=TAKdbjoDvIqKdLn>

(This technology is brought to you by Compute Canada and WestGrid with support from PHSA Telehealth)

