# Oct - Chris Hogue

**Speaker**: Chris Hogue

**Title of talk:** Information processing for the verification of physical molecular interactions.

!!! info "Event Details"
    
    
    **Date:**
    
    October 14, 2004

**Affiliation:** [The Blueprint Initiative](http://www.blueprint.org/)

[Samuel Lunenfeld Research Institute, Toronto](http://www.mshri.on.ca/)

**URL:** <http://www.blueprint.org/research/research_pi.html>

**Abstract:**

Mechanistic knowledge has been transformed from static pictures of pathways and reactions to new databases of interactions, complexes and pathways. This information is critically important for the further elucidation of the mechanisms of life and disease, yet it is filled with experimental uncertainties and high rates of false information. We have recently been trying to untangle the possible physical interactions in S. cerevisiae starting from high-throughput experiments defining molecular complexes and interactions from tagged pull-down and yeast-two-hybrid methods. Four areas of informatics research have begun to bear fruit. First, we have undertaken to analyze and report domain-domain correlations to better understand the underlying makeup of protein complexes. When protein complex networks are simplified into domain-domain correlations, the overlap between different experimental datasets improves. Second, we have developed methods to define and predict specific domain-motif interactions in protein complexes in yeast. In particular, more domain-motif interactions are found in low-affinity purification methods (FLAG-tag) than in high-affinity methods (TAP-tag), although this observation may be bait-biased. Third, we have been attempting to build tools to understand and annotate the physical interconnections between protein domains and their small molecule binding sites through the Small Molecule Interaction Database (SMID). SMID works with and a variant of RPS-BLAST, enhanced to locate small molecule binding sites based on PDB 3D structure and NCBI CDD information. And finally, we have developed a SVM based score called BIND-PICKS (Protein Interaction Confidence Kernel Scores) which will form the basis for reporting on the quality of yeast physical interactions. About 18% of all high-throughput data (complexes as spoke models) scores above the false positive threshold with BIND-PICKS. Detailed comparisions with specific data sets and protein interaction prediction methods will be discussed.

—–

---

**Student Speaker:** Stephen Montgomery – Michael Smith Genome Sciences Centre

**Title of talk:** Chinook: Acollaborative system for bioinformatics analysis.

**Affiliation:** [Michael Smith Genome Sciences Centre](http://www.bcgsc.ca/)

**URL:** <http://smweb.bcgsc.bc.ca/>

**Presentation:** [Download :material-file-pdf-box:](https://drive.google.com/file/d/1BpFuBl3gMAU4GBv1sm2FBtOjrbJulgzg/view?usp=sharing) [:material-file-powerpoint-box:](https://drive.google.com/file/d/1paQwVUZDxQxR3xAL-fFojuofA051UGBh/view?usp=sharing)

[![Creative Commons License](http://i.creativecommons.org/l/by-sa/3.0/80x15.png)](http://creativecommons.org/licenses/by-sa/3.0/)

This work is licensed under a [Creative Commons Attribution-Share Alike 3.0 License](http://creativecommons.org/licenses/by-sa/3.0/).

