# Sep - Gabor Marth

**Speaker**: Gabor Marth

**Title of talk:** The utility of the HapMap reference samples for clinical populations

!!! info "Event Details"
    
    
    **Date:**
    
    September 9th, 2004

**Affiliation:** Boston College

**URL:** <http://www.bc.edu/schools/cas/biology/facadmin/marth/>

**Presentation:** [Download :material-file-pdf-box:](https://drive.google.com/file/d/1k8vfc44U6SleMJBLFzr77didt1a_60sU/view?usp=sharing) [:material-file-powerpoint-box:](https://drive.google.com/file/d/1qldQAuln92GVH4J7wlqHRGgmpxREZyRF/view?usp=sharing)

**Abstract:**

A kilobase-scale map of human polymorphism structure, the HapMap, is under construction to help us understand the molecular and demographic forces that have shaped the variation landscape of our species and to identify from millions of polymorphic sites the small subset with functional effects. Because the mapping of disease-causing variants requires the coinheritance of the marker and the functional allele, the main motivation for the HapMap is to assess the strength of allelic association along human chromosomes. This is accomplished by genotyping, at millions of polymorphic sites, sets of reference samples from a handful of large world populations, and annotating measures of linkage disequilibrium (LD) and haplotype diversity within these samples.

Although most polymorphisms are shared among all human populations, there are significant differences in their allele frequencies. Additionally, the strength of allelic association may not only vary between samples of different ethnic origins, but also across consecutive sets drawn from even a single population. Therefore, the utility of the HapMap resource for clinical association studies will largely depend on the degree to which the annotations based on the reference individuals are generalizable for consecutive, clinical samples.

In this talk we will discuss the basic concepts and algorithms used in the analysis of multi-locus haplotypes: experimental haplotype determination, statistical methods of haplotype inference / reconstruction from diploid genotype data, pair-wise LD-based and haplotype-based measures of allelic association, haplotype and LD block extraction. We will show how SNP discovery and the selection of SNPs for inclusion into the HapMap have shaped the sampling properties of this extensive and expensive genotype resource. Finally, along the lines of our primary research interests, we demonstrate how quantitative models of long term demographic history, a major determinant of differential allele structure among geographically subdivided populations, can be used both to analyze the association properties of the HapMap reference samples, and to predict the strength of allelic association for additional, future samples, such as those that will be used in clinical case-control studies.

â€”

---

**Student Speaker:** Jenn Gardy

**Title of talk:** Updates from the Protein Localization Prediction Front: PSORTb, PSORTdb and Perspectives on Predictive Methods

**Affiliation:** SFU Brinkman Laboratory

**URL:** <http://www.pathogenomics.sfu.ca/brinkman/index.html>

**Presentation:** [Download :material-file-pdf-box:](https://drive.google.com/file/d/1Ffwetzhgu81LfDMy1DfwK0xov35diGOA/view?usp=sharing) [:material-file-powerpoint-box:](https://drive.google.com/file/d/1j4P2QOz58Akxw35x_i4rolfL8ZbeMvEd/view?usp=sharing)

[![Creative Commons License](http://i.creativecommons.org/l/by-sa/3.0/80x15.png)](http://creativecommons.org/licenses/by-sa/3.0/)

This work is licensed under a [Creative Commons Attribution-Share Alike 3.0 License](http://creativecommons.org/licenses/by-sa/3.0/).

